osf_tests/test_maintenance.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from datetime import timedelta
+
+import pytest
+from django.utils import timezone
+
+from website import maintenance
+from osf.models import MaintenanceState
+
+
+pytestmark = pytest.mark.django_db
+
+
+class TestMaintenance(unittest.TestCase):
```

Comment:
Since we're using pytest now, we no longer need to subclass `TestCase`. You  can still use a [teardown method](https://docs.pytest.org/en/latest/xunit_setup.html?highlight=teardown#method-and-function-level-setup-teardown).

```python
class TestMaintenance:
    def teardown_method(self, method):
         #...
```